homeassistant/components/light/osramlightify.py

```diff
@@ -96,7 +94,7 @@ def __init__(self, light_id, light, update_lights):
         self._brightness = 0
         self._rgb = (0, 0, 0)
         self._name = ""
-        self._temperature = TEMP_MIN
+        self._temperature = 2000
```

Comment:
While we're doing cleanup here, can you just change `self._brightness`, `self._rgb`, `self._name`, `self._temperature`, and `self._state` assignments in `__init__` to `None`? These will get overwritten when `self.update()` is called, so it's safer/cleaner to initialize them to `None` since it shouldn't matter if everything is working.

Also if you're feeling extra ambitious you can drop the `self.update()` call in `__init__`, and change `add_devices_callback(new_lights)` to `add_devices_callback(new_lights, True)` to migrate to the new style and remove IO from the constructor.